README.md

```diff
@@ -62,27 +62,25 @@ you can use `[headers]` directive like this.
 
 #### Callbacks
```

Comment:
While we're here - though it's not strictly incorrect, I would rename this to either "Outputs" or "Events".